### ISSUE TITLE
DDPG Critic implementation

### ISSUE THREAD

**Issue by dynamik1703** [MorvanZhou/Reinforcement-learning-with-tensorflow#49]
Hi MorvanZhou,
I have a question about implementation of the critic in the ddpg.

In the baseline implementation is a difference:
https://github.com/openai/baselines/blob/master/baselines/ddpg/models.py

class Critic(Model):
...
x = obs
            x = tf.layers.dense(x, 64)
...
            x = tf.nn.relu(x)

            x = tf.concat([x, action], axis=-1)
            x = tf.layers.dense(x, 64)
...
            x = tf.nn.relu(x)
...
The action is inserted via concat in the second layer. 

This differs from the implementation in this repo. There the action is expanded in the range of n_l1 and is finally summed up in the first layer.
#            w1_s = tf.get_variable('w1_s', [self.s_dim, n_l1], trainable=trainable)
#            w1_a = tf.get_variable('w1_a', [self.a_dim, n_l1], trainable=trainable)
#            b1 = tf.get_variable('b1', [1, n_l1], trainable=trainable)
#            net = tf.nn.relu(tf.matmul(s, w1_s) + tf.matmul(a, w1_a) + b1)

What do you think is the correct implementation?

Thanks,
Roman
